appveyor.yml

```diff
@@ -30,16 +31,18 @@ environment:
     C:\\Program Files\\Git\\cmd;\
     C:\\Program Files\\Git\\usr\\bin;\
     C:\\Program Files\\AppVeyor\\BuildAgent;"
+
   CC: "clang-cl.exe"
   CXX: "clang-cl.exe"
-  LINKER: "lld-link.exe"
 
   VCVARSALL_PATH: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build
 
   matrix:
     - TARGET: i686-pc-windows-msvc
       CROSS_COMPILE: 1
     - TARGET: x86_64-pc-windows-msvc
+    - TARGET: x86_64-pc-windows-msvc
```

Comment:
Should we add some UWP targets too?

Reply:
Can't without using something like xargo to also build the standard library, sadly.